README.md

```diff
@@ -25,9 +25,11 @@ gem 'twingly-search'
 require 'twingly/search'
 
 client = Twingly::Search::Client.new
```

Comment:
*late to the part* we could append their own user agent and always include the part we want